test/kitchen-automate/README.md

```diff
@@ -0,0 +1,49 @@
+# kitchen-automate Cookbook
+
+Integration testing for the `audit` cookbook.
+
+## Purpose of this cookbook
+
+Tests the audit cookbook for the `chef-visibility` and `chef-server-visibility` collectors with a node managed by a Chef Server that is integrated with Chef Automate.
+
+## Requirements
+
+ * `test-kitchen` installed with `kitchen-ec2` driver.
+ * ENV variables `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` in order to create and destroy the test ec2 instance.
+ * Access to the private SSH key(`~/.ssh/kitchen-key-compliance.pem`) used in `.kitchen.yml`
+
+## How is it working?
+
+This cookbook uses the `kitchen-ec2` driver in order to spin up an OpsWorks(Chef Server + Automate) instance in ec2 using a previously snapshotted OpsWorks instance.
```

Comment:
Should we add `kitchen-ec2` to our Gemfile? https://github.com/chef-cookbooks/audit/blob/master/Gemfile#L23-L27